App.js

```diff
@@ -1,96 +1,11 @@
+import React from 'react'
 import { BASE_URL } from '@env'
 import init from 'startupjs/init'
+import Main from './components/Main'
 import orm from './model'
-import React, { useState, useEffect } from 'react'
-//import offlineInitPlugin from '@startupjs/offline/lib/offlineInitPlugin'
-import {
-  View,
-  Text,
-  TouchableOpacity
-} from 'react-native'
-import {
-  observer,
-  useDoc,
-  useApi,
-  $root
-} from 'startupjs'
-import axios from 'axios'
-
-import './App.styl'
-
-// Init startupjs connection to server and the ORM.
-// baseUrl option is required for the native to work - it's used
-// to init the websocket connection and axios.
-// Initialization must start before doing any subscribes to data.
+import { observer } from 'startupjs'
 init({ baseUrl: BASE_URL, orm })
-//init({ baseUrl: BASE_URL, orm, plugins: [offlineInitPlugin] })
 
 export default observer(function App () {
-  let [counter, $counter] = useDoc('counters', 'first')
-  if (!counter) throw $counter.addSelf() // custom ORM method (see /model/)
-
-  let [state] = useDoc('$connection', 'state')
-  let [offlineState] = useDoc('$connection', 'offlineState')
-
-  let [stateCounter, setStateCounter] = useState(0)
-
-  let forceTrigger = useForceTrigger(3000)
-  let [api] = useApi(getApi, [forceTrigger])
-
-  async function increment () {
-    $counter.increment('value', 1)
-    setStateCounter(stateCounter + 1)
-  }
-
-  async function decrement () {
-    $counter.increment('value', -1)
-    setStateCounter(stateCounter - 1)
-  }
-
-  async function reset () {
-    $counter.reset() // custom ORM method (see /model/)
-    setStateCounter(0)
-  }
-
-  return pug`
-    View.body
-      Text.greeting Hello World
-      Text Online status: #{state}
-      Text Online status (detailed): #{offlineState}         
-      Text DB Counter: #{counter && counter.value}
-      Text State Counter: #{stateCounter}
-      TouchableOpacity.button.increment(onPress=increment)
-        Text.label +
-      TouchableOpacity.button.decrement(onPress=decrement)
-        Text.label -
-      TouchableOpacity.button.clear(onPress=reset)
-        Text.label RESET
-      Text.api /api (updated each 3 sec): #{JSON.stringify(api)}
-  `
+  return(<Main/>)
```

Comment:
@TimmyByDeOverly please try to use pug. Example here https://github.com/TimmyByDeOverly/StartupJS/pull/1/files#diff-958f6f56cd42c904603d226cd36a57b6L55

Reply:
done